src/base/torrentfileswatcher.cpp

```diff
@@ -158,8 +158,8 @@ namespace
         QJsonObject jsonObj {
             {PARAM_CATEGORY, params.category},
             {PARAM_TAGS, serializeTagSet(params.tags)},
-            {PARAM_SAVEPATH, params.savePath},
-            {PARAM_DOWNLOADPATH, params.downloadPath},
+            {PARAM_SAVEPATH, params.savePath.toString()},
+            {PARAM_DOWNLOADPATH, params.downloadPath.toString()},
```

Comment:
If I read correct, those will be write to file on disk, right? And since the file is human readable (or even editable) then it should show the correct path separator.

Reply:
> If I read correct, those will be write to file on disk, right? And since the file is human readable (or even editable) then it should show the correct path separator.

Well, in this case, apparently you're right. But at a minimum, this will require additional tests to make sure that it does not break, since it wrote with uniform separators initially. And it will definitely break the possibility of a "painless" downgrade.

Reply:
>since it wrote with uniform separators initially

If uniform separators were used from the start, then just keep it as is. No need to break things now...
Maybe a comment a code saying this is intended would help developers in the future.

Reply:
> If uniform separators were used from the start, then just keep it as is. No need to break things now...

It will most likely break downgrade.
We can provide additional patch to v4.4.1 that will make it separator independent.